host-bin/enter-chroot

```diff
@@ -133,6 +136,13 @@ elif [ -n "$TARGET" ]; then
     fi
 fi
 
+# Check to ensure that the XMETHOD requested has been installed
+if [ -n "$TMPXMETHOD" ]; then
+    if ! grep -q "^$TMPXMETHOD$" "$CHROOTS/$NAME/.crouton-targets" 2>/dev/null; then
```

Comment:
I can definitely do that but I wasn't certain that $CHROOT was valid until the routine at line 166 '# Mount the chroot and update our CHROOT path' executed.